pyblish_qml/models.py

```diff
@@ -384,9 +384,9 @@ def add_instance(self, instance):
         item["hasCompatible"] = True
 
         # Visualised in Perspective
-        item["familiesConcatenated"] = instance["data"].get("family", "")
-        item["familiesConcatenated"] += ", ".join(
-            instance["data"].get("families", []))
+        families = [instance["data"]["family"]]
```

Comment:
`family` is guaranteed to exists since 1.2.1 and normally I would say we should keep backwards compatibility across the entire 1.0 spectrum; but Pyblish QML has already disavowed the `repair` functionality since a few versions back, so in this case I'll accept this.

Could you add a note to the README about compatibility with 1.2.1 and above?